### PR TITLE
Add an example code for `map.removeAll` method [API-15]

### PR DESCRIPTION
**Pull request by fatihozer0**

### CODE REVIEW

File: code_samples/map_removeAll_sample.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+'use strict';
```
Comment (srknzl) [inline code_samples/map_removeAll_sample.js:1]: Can you add copyright? See other files

File: code_samples/map_removeAll_sample.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+const { Client, Predicates } = require('../lib/index');
```
Comment (srknzl) [inline code_samples/map_removeAll_sample.js:3]:
```suggestion
const { Client, Predicates } = require('hazelcast-client');
```

**Review by srknzl — CHANGES_REQUESTED** [review]
some minor commetns are added

File: code_samples/map_removeAll_sample.js (hunk 1 above)
Comment (fatihozer0) [reply code_samples/map_removeAll_sample.js:1]: I would add it but could not be sure should I copy/paste the copyright or is there a way to do it?

File: code_samples/map_removeAll_sample.js (hunk 2 above)
Comment (fatihozer0) [reply code_samples/map_removeAll_sample.js:3]: Yes, my bad. Just wanted to see if it works

File: code_samples/map_removeAll_sample.js (hunk 1 above)
Comment (srknzl) [reply code_samples/map_removeAll_sample.js:16]: Just copy paste

**Review by srknzl — APPROVED** [review]

**Pull request merged**